File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        s = ""
        if len(strs)== 0: return s
        strs.sort()
        f = strs[0]
        l = strs[-1]
        for i in range(len(f)):
            if f[i] == l[i]:
                s +=f[i]
            else :
                break    
        return s        

        

File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] arr) {
        if (arr.length == 0) return "";
        Arrays.sort(arr);
        String f = arr[0];
        String l = arr[arr.length-1];
        String s = "";
        for(int i = 0; i < f.length(); i++){
            if(f.charAt(i)== l.charAt(i)){
                s+= f.charAt(i);
            }else{
                break;
            }
        }
        return s;
    }
}